test/tic_tac_toe/ai_test.clj

```diff
@@ -192,4 +194,49 @@
   (let [player  (->AiPlayer "o")
         game { :board [["_" "_" "_" ]["_" "_" "_" ]["_" "_" "_" ]] :ai-marker "o" :player-marker "x"}]
     (testing "creates defrecord of player protocol"
-      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
\ No newline at end of file
+      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
+
+(defn win-or-draw? [player-marker winner-state]
+ (or (= player-marker winner-state) (= player-marker "its a draw")))
```

Comment:
Asserting against raw strings make your tests very fragile. If I changed to prompt to have proper capitalization, these tests would break and it would not be very clear to anyone why.